src/test/java/api/requests/scenarios/MoveRequestTests.java

```diff
@@ -697,6 +697,80 @@ public void canMoveARecallRequestAsHoldRequest()
     assertThat(interestingTimesQueue.getTotalRecords(), is(2));
   }
 
+  @Test
+  public void cannotDisplacePagedRequest()
+      throws MalformedURLException,
+      InterruptedException,
+      TimeoutException,
+      ExecutionException {
+
+      final IndividualResource secondFloorEconomics = locationsFixture.secondFloorEconomics();
+      final IndividualResource mezzanineDisplayCase = locationsFixture.mezzanineDisplayCase();
+
+      final IndividualResource itemCopyA = itemsFixture.basedUponTemeraire(
```

Comment:
To prove this feature working, do the items have to be copies of the same title or can they be from any title?

Reply:
The reported scenario utilized copies of the same title, but this should work in either case.

Reply:
Should we add a similar test case for different titles? What do you think? Is it redundant or is there any value? I see copies (items) and titles (instances) as separate concepts that need to be documented either by adding another test case or a comment to say that this test case should work for instances as well.

Reply:
I will investigate the logic and see if such a test is warranted.

Reply:
I attempted the tests with two disparate instances and it passes. I have also reviewed the mechanisms involved in moving a request and see no reason that they would behave differently for items of the same or different instance. I added a comment to the test conveying this.

Reply:
looks good, thx.